build/pubspec.yaml

```diff
@@ -21,6 +21,12 @@ dependencies:
   watcher: ^0.9.7
   yaml: ^2.1.0
 
+dependency_overrides:
```

Comment:
These are needed for travis tests to pass locally. I didn't realize you can't publish with them... Removing